draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -317,19 +312,17 @@ consumer(s) how to treat DNSSEC for the zones "example.net." and "example.com.":
 
 ```
 <unique-1>.zones.$CATZ        0 IN PTR    example.com.
-group.<unique-1>.zones.$CATZ  0 IN TXT    "nodnssec"
+group.<unique-1>.zones.$CATZ  0 IN TXT    "foo"
 <unique-2>.zones.$CATZ        0 IN PTR    example.net.
-group.<unique-2>.zones.$CATZ  0 IN TXT    "operator-x-sign-with-nsec3"
-group.<unique-2>.zones.$CATZ  0 IN TXT    "operator-y-nsec3"
+group.<unique-2>.zones.$CATZ  0 IN TXT    "operator-x-foo"
+group.<unique-2>.zones.$CATZ  0 IN TXT    "operator-y" "bar"
```

Comment:
Is this admissible? Further up, we say: `A `group` property value is stored as content of a TXT record`

Here it seems we're storing two such values in one record (as opposed to having two records at the same owner name). I recall @Habbie saying in the past that there's only one such value per record.

I have no stakes at all, just raising the point in case it is a mistake.

Reply:
I noticed this too, but I interpreted it differently as "operator-y" "bar" being one group property value. Since we had no previous text disallowing more than one TXT rdata field, I thought it was a good clarification to use as an example too.

Reply:
It is intentional, the last example is there to point out that any value that fits in TXT will do. The draft has no opinion about what the producer and consumer operators agree on.

Reply:
I think that's confusing. I would like to suggest to restrict group property values to a single character string (in the sense of [RFC 1035 Section 3.3.14](https://www.rfc-editor.org/rfc/rfc1035#section-3.3.14)), unless there's a usecase for doing the opposite.

Actually, I think it would be need to specify that *any* character string in the TXT RRset represents one group property value, regardless of whether they're in one or several records. Considering that, all of these three would be equivalent:

```
; case 1
group.<unique-2>zones.$CATZ  0 in TXT   "operator-y" "bar"
group.<unique-2>zones.$CATZ  0 in TXT   "baz"

; case 2
group.<unique-2>zones.$CATZ  0 in TXT   "baz" "operator-y" "bar"

; case 3
group.<unique-2>zones.$CATZ  0 in TXT   "operator-y"
group.<unique-2>zones.$CATZ  0 in TXT   "bar"
group.<unique-2>zones.$CATZ  0 in TXT   "baz"
```

I think this is the approach of least confusion while still covering all use cases (AFAIK).

Reply:
> It is intentional, the last example is there to point out that any value that fits in TXT will do. The draft has no opinion about what the producer and consumer operators agree on.

Fair; as I said, I have no stakes, but confusion should be avoided. If this is the intent, then in line 288 let's replace
```
A `group` property value is stored as content of a TXT record directly below the member node.
```
by
```
A `group` property value is stored as the entire Rdata of a TXT record directly below the member node.
```
to make it fully clear. (I think the word `content` is too ambiguous, because people could mis-take it to mean a character string only.)

Reply:
I don't like it that way @peterthomassen because it would make properties less generic. Now it is one RR one property value. Also, we had restrictions on group property values removed earlier in commit 718d25d2

Reply:
:) I think we're interlacing replies. I do support your suggestion to rephrase "content" with "the entire Rdata".